src/project.test.js

```diff
@@ -0,0 +1,20 @@
+const starBorder = document.createElement('span');
+const starCount = document.createElement('span');
+const postLike = require('./modules/involvement.js');
+const updateLikes = require('./modules/involvement.js');
+
+function showLikes() {
+  starBorder.addEventListener('click', () => {
+    postLike();
```

Comment:
- [ ] Good job writing a unit test for the show likes function. Please write unit tests for the movie counter feature of your app. Your page shows the number of movies in the homepage. Please write unit tests for that counter method.